README.md

```diff
@@ -106,8 +106,6 @@ For more information, please refer to the developer documentation of these servi
 
 For further information on OAuth endpoints, see the Internet Engineering Task Force (IETF) RFC 6749, [The OAuth 2.0 Authorization Framework](https://tools.ietf.org/html/rfc6749), page 21.
 
-*Note*: When setting Chef Automate as your OAuth provider, you will need to add your Automate instance's TLS certificate (found at the `load_balancer.v1.sys.frontend_tls` entry in your Chef Automate `config.toml` file), to your Chef Habitat Builder on-prem instance's list of accepted certs. This can be done by copying the certificate to the `\hab\cache\ssl` folder on the Chef Habitat Builder on-prem instance (supported by the latest version of the Chef Habitat Builder on-prem), or alternatively by modifying the `core/cacert` package and appending the cert to the cert.pem file at the following location: `$(hab pkg path core/cacerts)/ssl/cert.pem`.
```

Comment:
We don't want to entirely remove this paragraph - just the last part that says:

```
or alternatively by modifying the `core/cacert` package and appending the cert to the cert.pem file at the following location: `$(hab pkg path core/cacerts)/ssl/cert.pem`.
```

Reply:
I'm moving it onto the Automate page because it doesn't make a lot of sense in this example.

Reply:
ok cool